driver/usb_hid_keys.h

```diff
@@ -0,0 +1,29 @@
+// USB HID keycodes
+// https://source.android.com/devices/input/keyboard-devices.html
+#pragma once
```

Comment:
Remove the `#pragma once` as you have the `#ifndef` below as header guard

Reply:
'pragma once' increases compiling speed and works better IDE's code completion, but it may be not implemented in some compilers.

I have used both (pragma and ifndef-define) as a compromise between "works everywhere" and "works better".